bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/BagVerifierWorkerService.scala

```diff
@@ -3,31 +3,21 @@ package uk.ac.wellcome.platform.archive.bagverifier.services
 import akka.Done
 import grizzled.slf4j.Logging
 import org.apache.commons.codec.digest.MessageDigestAlgorithms
-import org.slf4j.MDC
-import uk.ac.wellcome.json.JsonUtil._
-import uk.ac.wellcome.messaging.sns.{PublishAttempt, SNSWriter}
 import uk.ac.wellcome.messaging.sqs.NotificationStream
 import uk.ac.wellcome.platform.archive.bagverifier.models.BagVerification
 import uk.ac.wellcome.platform.archive.common.models.BagRequest
 import uk.ac.wellcome.platform.archive.common.models.bagit.BagLocation
-import uk.ac.wellcome.platform.archive.common.progress.models.{
-  Progress,
-  ProgressEvent,
-  ProgressStatusUpdate,
-  ProgressUpdate
-}
 import uk.ac.wellcome.typesafe.Runnable
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
 class BagVerifierWorkerService(
-  notificationStream: NotificationStream[BagRequest],
-  verifyDigestFilesService: VerifyDigestFilesService,
-  progressSnsWriter: SNSWriter,
-  outgoingSnsWriter: SNSWriter,
-)(implicit ec: ExecutionContext)
-    extends Runnable
+                                notificationStream: NotificationStream[BagRequest],
+                                verifyDigestFilesService: VerifyDigestFilesService,
+                                notificationService: NotificationService
```

Comment:
Naming nit: having a “notificationStream” that’s an input and a “notificationService” that’s an output feels liable to get confusing. Could we rename them to emphasise the input/output nature?